app/src/main/java/io/pslab/models/PSLabSensor.java

```diff
@@ -88,8 +88,10 @@
     public final String DATA_BLOCK = "data_block";
 
     public static final String LUXMETER = "Lux Meter";
+    public static final String LUXMETER_CONFIGURATIONS = "Lux Meter Configurations";
```

Comment:
Do we need this line in `PSLabSensor` file? I can't see its use anywhere

Reply:
Those are for the switch cases :)

Reply:
Oh! Just missed that..